diff -burN rogue5.4.4-ant-r1.1.2.orig/main.c rogue5.4.4-ant-r1.1.2/main.c
--- rogue5.4.4-ant-r1.1.2.orig/main.c	2014-12-18 02:08:50.000000000 +0000
+++ rogue5.4.4-ant-r1.1.2/main.c	2016-08-31 03:17:05.894848200 +0000
@@ -276,8 +276,12 @@
   getyx(curscr, y, x);
   mvcur(y, x, oy, ox);
   fflush(stdout);
+#ifdef NCURSES_OPAQUE
+  wmove(curscr, oy, ox);
+#else
   curscr->_cury = oy;
   curscr->_curx = ox;
+#endif /* NCURSES_OPAQUE */
 }
 
 /*
diff -burN rogue5.4.4-ant-r1.1.2.orig/mdport.c rogue5.4.4-ant-r1.1.2/mdport.c
--- rogue5.4.4-ant-r1.1.2.orig/mdport.c	2008-05-07 05:38:10.000000000 +0000
+++ rogue5.4.4-ant-r1.1.2/mdport.c	2016-08-31 03:23:42.181109500 +0000
@@ -128,8 +128,14 @@
 #endif
 
 #if defined(HAVE_ESCDELAY) || defined(NCURSES_VERSION)
+# ifdef NCURSES_OPAQUE
+  setenv("ESCDELAY", "64", 0 /* don't overwrite */);
+# else
   ESCDELAY=64;
-#endif
+# endif /* NCURSES_OPAQUE */
+#else
+  ESCDELAY=64;
+#endif /* HAVE_ESCDELAY || NCURSES_VERSION */
 
 #if defined(DUMP)
   md_onsignal_default();
